src/Network/Cookie/ResponseCookieJar.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Cake\Network\Cookie;
+
+class ResponseCookieJar
+{
+    protected $_cookies = [];
+
+    public function get($name) {
+        return $this->_cookies[$name];
+    }
+    
+    public function create($name, $config = []) {
+        $cookie = new Cookie($name, $config);
+
+        // $this->add($cookie) ???
+
+        return $cookie;
+    }
+
+    public function add(Cookie $cookie) {
```

Comment:
add() and create() don't seem that different. Why not have one method that accepts multiple types of arguments.